sw/device/lib/meson.build

```diff
@@ -113,3 +113,18 @@ sw_lib_usb = declare_dependency(
     ]
   )
 )
+
+# OTBN driver library (sw_lib_otbn)
+sw_lib_otbn = declare_dependency(
```

Comment:
I'm trying to empty out the device/lib directory, since it's kind of a kitchen sink right now. Maybe we should pick some other place for it to go in? I would suggest lib/runtime, but @lenary and I are worried about it *also* becoming a kitchen sink.

Reply:
Let me know when there's an actionable proposal for a new location and I'll happily move things there.

Reply:
Sure, done.